src/app/home/components/shared/Section.tsx

```diff
@@ -0,0 +1,29 @@
+import SectionHeader from './SectionHeader'
+import * as styles from '@/app/home/components/shared/Section.css'
+
+interface SectionProps {
+  title: string
+  children: React.ReactNode
+  showRight?: boolean
+  showNotice?: boolean
+}
+
+const Section = ({
+  title,
+  children,
+  showRight = false,
+  showNotice = false,
+}: SectionProps) => {
+  return (
+    <div>
+      <SectionHeader
+        text={title}
+        showRight={showRight}
+        showNotice={showNotice}
+      />
+      <div className={styles.sectionContent}>{children}</div>
+    </div>
```

Comment:
```suggestion
    <>
      <SectionHeader
        text={title}
        showRight={showRight}
        showNotice={showNotice}
      />
      <div className={styles.sectionContent}>{children}</div>
    </>
```
request: 단순히 묶는 용도라면 빈 태그가 좋을 것 같아용

Reply:
div 태그를 사용하면 불필요한 노드가 추가되기 때문에 단순히 묶는 용도라면 <></>를 사용하는 것이 좋아요~
이 의도로 질문주신거 맞을까요??

Reply:
네 맞아요!! 제가 검색해봤는데 둘의 차이점을 검색해도 잘 못찾겠어서 ... 궁금증이 완벽하게 해결되었어요 👍 👍 🥹

Reply:
정식 명칭은 fragment라서 요걸로 검색해야 잘 나올 것 같네용 공식문서 링크 남겨요 참고참고

https://ko.react.dev/reference/react/Fragment